tests/sessions_tests/tests.py

```diff
@@ -353,6 +354,27 @@ def test_session_load_does_not_create_record(self):
         # provided unknown key was cycled, not reused
         self.assertNotEqual(session.session_key, 'someunknownkey')
 
+    def test_session_save_does_not_resurrect_session_logged_out_in_other_context(self):
+        """ Ensure that sessions are not resurrected by a concurrent request. """
+        if self.backend is CookieSession:
+            raise unittest.SkipTest("CookieSessions is stored in the client and there is no way to query it.")
```

Comment:
Define a `CookieSessionTests.test_session_save_does_not_resurrect_session_logged_out_in_other_context` method decorated with `unittest.skip` instead to keep `SessionTestsMixin` backend agnostic. See #6203.